app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
+# frozen_string_literal: true
```

Comment:
Add to `rubicon.yml`
Style/Documentation:
  Enabled: false

Metrics/ClassLength:
  Enabled: false

Style/FrozenStringLiteralComment:
  Enabled: false

to avoid unnecessary recommendations from rubocop